v22.1/import.md

```diff
@@ -315,14 +315,23 @@ You can also use the [`cockroach nodelocal upload`](cockroach-nodelocal-upload.h
 
 ### Import data into your {{ site.data.products.db }} cluster
 
-{{site.data.alerts.callout_info}}
-For {{ site.data.products.serverless }} clusters, you must have [billing information](../cockroachcloud/billing-management.html) on file for your organization to have access to [cloud storage](../{{site.current_cloud_version}}/use-cloud-storage-for-bulk-operations.html). If you don't have billing set up, [`userfile`](../{{site.current_cloud_version}}/use-userfile-for-bulk-operations.html) is your **only available storage option** for bulk operations. {{ site.data.products.dedicated }} users can run bulk operations with `userfile` or cloud storage.
-{{site.data.alerts.end}}
+You can import data into your {{ site.data.products.db }} cluster using either [`userfile`](use-userfile-for-bulk-operations.html) or [cloud storage](use-cloud-storage-for-bulk-operations.html):
+
+<div class="filters clearfix">
+  <button class="filter-button" data-scope="userfile"><code>userfile</code></button>
+  <button class="filter-button" data-scope="cloud">Cloud storage</button>
+</div>
 
-#### Import using a userfile
+<section class="filter-content" markdown="1" data-scope="userfile">
+
+#### Import using `userfile`
 
 {% include cockroachcloud/userfile-examples/import-into-userfile.md %}
 
+</section>
+
+<section class="filter-content" markdown="1" data-scope="cloud">
+
 #### Import using Cloud storage
```

Comment:
```suggestion
#### Import using cloud storage
```

Reply:
This wasn't your work, but "Cloud" on the title here can be lowercased "Import using cloud storage"